include/rcutils/filesystem.h

```diff
@@ -22,6 +22,13 @@ extern "C"
 
 #include <stdbool.h>
 #include <stddef.h>
+#include <stdint.h>
+#ifndef _WIN32
+#include <sys/types.h>
+#else
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
+#endif
```

Comment:
Since we aren't using `ssize_t` anymore, do we still need this block?

Reply:
Yeah. Above codes should be removed.